src/task 31/main.js

```diff
@@ -0,0 +1,44 @@
+//https://jsonplaceholder.typicode.com/users
+//https://eloquentjavascript.net/author
+var requestURL = "https://eloquentjavascript.net/author"
+var requestTypes = ["text/plain",
+                      "text/html",
+                      "application/json",
+                      "application/rainbows+unicorns"];
+
+function sendRequest(method, url, body = null){ 
+    return new Promise(function(resolve, reject){
+        for(key of requestTypes){
```

Comment:
It's more natural to use `forEach` method to iterate through the defined array of strings.